pkg/handlers/search/delete_by_query_test.go

```diff
@@ -0,0 +1,159 @@
+package search
+
+import (
+	"github.com/goccy/go-json"
+	"github.com/stretchr/testify/assert"
+	"github.com/zinclabs/zinc/pkg/core"
+	"github.com/zinclabs/zinc/pkg/ider"
+	"github.com/zinclabs/zinc/pkg/meta"
+	"github.com/zinclabs/zinc/test/utils"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+type arg struct {
+	doc    map[string]interface{}
+	query  string
+	params map[string]string
+}
+type body struct {
```

Comment:
you should add a blank line after each define end. 

also it can use format code.